src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -3828,6 +3828,7 @@ void Lowering::ContainCheckHWIntrinsic(GenTreeHWIntrinsic* node)
             case NI_Sve_AddRotateComplex:
             case NI_Sve_TrigonometricMultiplyAddCoefficient:
             case NI_Sve2_ShiftLeftAndInsert:
+            case NI_Sve2_XorRotateRight:
```

Comment:
i don't think we need this as well. It should get handled in `intrin.category == HW_Category_ShiftRightByImmediate` condition above.